bsp/bsp.h

```diff
@@ -29,6 +29,43 @@ enum {
   EXC_CAUSE_INSTR_INTEGRITY_FAULT = 25,
 };
 
+typedef enum {
+  PMPMODE_OFF   = 0,
+  PMPMODE_TOR   = 1,
+  PMPMODE_NA4   = 2,
+  PMPMODE_NAPOT = 3
+} pmp_mode_t;
```

Comment:
We agrees on aiming for similar bsp and so, so I think it is fine to have this pmp code.

Reply:
RESOLVED